2019-05-02T19:15:35.025-0700 I CONTROL  [initandlisten] MongoDB starting : pid=700 port=27017 dbpath=/vm/kevin/crunch/report/db/ 64-bit host=127x13.csc.calpoly.edu
2019-05-02T19:15:35.025-0700 I CONTROL  [initandlisten] db version v4.0.3
2019-05-02T19:15:35.025-0700 I CONTROL  [initandlisten] git version: 7ea530946fa7880364d88c8d8b6026bbc9ffa48c
2019-05-02T19:15:35.025-0700 I CONTROL  [initandlisten] OpenSSL version: OpenSSL 1.1.1b  26 Feb 2019
2019-05-02T19:15:35.025-0700 I CONTROL  [initandlisten] allocator: tcmalloc
2019-05-02T19:15:35.025-0700 I CONTROL  [initandlisten] modules: none
2019-05-02T19:15:35.025-0700 I CONTROL  [initandlisten] build environment:
2019-05-02T19:15:35.025-0700 I CONTROL  [initandlisten]     distarch: x86_64
2019-05-02T19:15:35.025-0700 I CONTROL  [initandlisten]     target_arch: x86_64
2019-05-02T19:15:35.025-0700 I CONTROL  [initandlisten] options: { net: { port: 27017 }, security: { javascriptEnabled: false }, storage: { dbPath: "/vm/kevin/crunch/report/db/", directoryPerDB: true }, systemLog: { destination: "file", path: "/vm/kevin/crunch/report/db/log.log", quiet: true } }
2019-05-02T19:15:35.025-0700 W STORAGE  [initandlisten] Detected unclean shutdown - /vm/kevin/crunch/report/db/mongod.lock is not empty.
2019-05-02T19:15:35.025-0700 I STORAGE  [initandlisten] Detected data files in /vm/kevin/crunch/report/db/ created by the 'wiredTiger' storage engine, so setting the active storage engine to 'wiredTiger'.
2019-05-02T19:15:35.025-0700 W STORAGE  [initandlisten] Recovering data from the last clean checkpoint.
2019-05-02T19:15:35.025-0700 I STORAGE  [initandlisten] 
2019-05-02T19:15:35.025-0700 I STORAGE  [initandlisten] ** WARNING: Using the XFS filesystem is strongly recommended with the WiredTiger storage engine
2019-05-02T19:15:35.025-0700 I STORAGE  [initandlisten] **          See http://dochub.mongodb.org/core/prodnotes-filesystem
2019-05-02T19:15:35.025-0700 I STORAGE  [initandlisten] wiredtiger_open config: create,cache_size=15443M,session_max=20000,eviction=(threads_min=4,threads_max=4),config_base=false,statistics=(fast),log=(enabled=true,archive=true,path=journal,compressor=snappy),file_manager=(close_idle_time=100000),statistics_log=(wait=0),verbose=(recovery_progress),
2019-05-02T19:15:35.945-0700 I STORAGE  [initandlisten] WiredTiger message [1556849735:945403][700:0x7f1a092419c0], txn-recover: Main recovery loop: starting at 3/1398656
2019-05-02T19:15:35.945-0700 I STORAGE  [initandlisten] WiredTiger message [1556849735:945707][700:0x7f1a092419c0], txn-recover: Recovering log 3 through 4
2019-05-02T19:15:36.005-0700 I STORAGE  [initandlisten] WiredTiger message [1556849736:4995][700:0x7f1a092419c0], txn-recover: Recovering log 4 through 4
2019-05-02T19:15:36.050-0700 I STORAGE  [initandlisten] WiredTiger message [1556849736:50773][700:0x7f1a092419c0], txn-recover: Set global recovery timestamp: 0
2019-05-02T19:15:36.055-0700 I RECOVERY [initandlisten] WiredTiger recoveryTimestamp. Ts: Timestamp(0, 0)
2019-05-02T19:15:36.172-0700 I CONTROL  [initandlisten] 
2019-05-02T19:15:36.172-0700 I CONTROL  [initandlisten] ** WARNING: Access control is not enabled for the database.
2019-05-02T19:15:36.172-0700 I CONTROL  [initandlisten] **          Read and write access to data and configuration is unrestricted.
2019-05-02T19:15:36.172-0700 I CONTROL  [initandlisten] 
2019-05-02T19:15:36.172-0700 I CONTROL  [initandlisten] ** WARNING: This server is bound to localhost.
2019-05-02T19:15:36.172-0700 I CONTROL  [initandlisten] **          Remote systems will be unable to connect to this server. 
2019-05-02T19:15:36.172-0700 I CONTROL  [initandlisten] **          Start the server with --bind_ip <address> to specify which IP 
2019-05-02T19:15:36.172-0700 I CONTROL  [initandlisten] **          addresses it should serve responses from, or with --bind_ip_all to
2019-05-02T19:15:36.172-0700 I CONTROL  [initandlisten] **          bind to all interfaces. If this behavior is desired, start the
2019-05-02T19:15:36.172-0700 I CONTROL  [initandlisten] **          server with --bind_ip 127.0.0.1 to disable this warning.
2019-05-02T19:15:36.172-0700 I CONTROL  [initandlisten] 
2019-05-02T19:15:36.172-0700 I CONTROL  [initandlisten] 
2019-05-02T19:15:36.172-0700 I CONTROL  [initandlisten] ** WARNING: /sys/kernel/mm/transparent_hugepage/enabled is 'always'.
2019-05-02T19:15:36.172-0700 I CONTROL  [initandlisten] **        We suggest setting it to 'never'
2019-05-02T19:15:36.172-0700 I CONTROL  [initandlisten] 
2019-05-02T19:15:36.172-0700 I CONTROL  [initandlisten] ** WARNING: /sys/kernel/mm/transparent_hugepage/defrag is 'always'.
2019-05-02T19:15:36.172-0700 I CONTROL  [initandlisten] **        We suggest setting it to 'never'
2019-05-02T19:15:36.172-0700 I CONTROL  [initandlisten] 
2019-05-02T19:15:36.274-0700 I FTDC     [initandlisten] Initializing full-time diagnostic data capture with directory '/vm/kevin/crunch/report/db/diagnostic.data'
2019-05-02T19:15:36.275-0700 I NETWORK  [initandlisten] waiting for connections on port 27017
2019-05-02T19:15:36.497-0700 I NETWORK  [conn1] end connection 127.0.0.1:35030 (0 connections now open)
2019-05-02T19:15:36.804-0700 I NETWORK  [conn2] received client metadata from 127.0.0.1:35032 conn2: { driver: { name: "PyMongo", version: "3.7.2" }, os: { type: "Linux", name: "Linux", architecture: "x86_64", version: "3.10.0-862.6.3.el7.x86_64" }, platform: "CPython 3.6.7.final.0" }
2019-05-02T19:15:36.805-0700 I NETWORK  [conn3] received client metadata from 127.0.0.1:35034 conn3: { driver: { name: "PyMongo", version: "3.7.2" }, os: { type: "Linux", name: "Linux", architecture: "x86_64", version: "3.10.0-862.6.3.el7.x86_64" }, platform: "CPython 3.6.7.final.0" }
2019-05-02T19:15:36.954-0700 I NETWORK  [conn4] received client metadata from 127.0.0.1:35036 conn4: { driver: { name: "PyMongo", version: "3.7.2" }, os: { type: "Linux", name: "Linux", architecture: "x86_64", version: "3.10.0-862.6.3.el7.x86_64" }, platform: "CPython 3.6.7.final.0" }
2019-05-02T19:15:36.955-0700 I NETWORK  [conn5] received client metadata from 127.0.0.1:35038 conn5: { driver: { name: "PyMongo", version: "3.7.2" }, os: { type: "Linux", name: "Linux", architecture: "x86_64", version: "3.10.0-862.6.3.el7.x86_64" }, platform: "CPython 3.6.7.final.0" }
2019-05-02T19:15:36.960-0700 I NETWORK  [conn3] end connection 127.0.0.1:35034 (3 connections now open)
2019-05-02T19:15:36.960-0700 I NETWORK  [conn2] end connection 127.0.0.1:35032 (2 connections now open)
2019-05-02T19:15:37.024-0700 I FTDC     [ftdc] Unclean full-time diagnostic data capture shutdown detected, found interim file, some metrics may have been lost. OK
2019-05-02T19:15:38.171-0700 I NETWORK  [conn6] received client metadata from 127.0.0.1:35040 conn6: { driver: { name: "PyMongo", version: "3.7.2" }, os: { type: "Linux", name: "Linux", architecture: "x86_64", version: "3.10.0-862.6.3.el7.x86_64" }, platform: "CPython 3.6.7.final.0" }
2019-05-02T19:15:38.172-0700 I NETWORK  [conn7] received client metadata from 127.0.0.1:35042 conn7: { driver: { name: "PyMongo", version: "3.7.2" }, os: { type: "Linux", name: "Linux", architecture: "x86_64", version: "3.10.0-862.6.3.el7.x86_64" }, platform: "CPython 3.6.7.final.0" }
2019-05-02T19:17:38.831-0700 I NETWORK  [conn7] end connection 127.0.0.1:35042 (3 connections now open)
2019-05-02T19:17:38.831-0700 I NETWORK  [conn6] end connection 127.0.0.1:35040 (2 connections now open)
2019-05-02T19:17:39.875-0700 I NETWORK  [conn8] received client metadata from 127.0.0.1:35048 conn8: { driver: { name: "PyMongo", version: "3.7.2" }, os: { type: "Linux", name: "Linux", architecture: "x86_64", version: "3.10.0-862.6.3.el7.x86_64" }, platform: "CPython 3.6.7.final.0" }
2019-05-02T19:17:39.876-0700 I NETWORK  [conn9] received client metadata from 127.0.0.1:35050 conn9: { driver: { name: "PyMongo", version: "3.7.2" }, os: { type: "Linux", name: "Linux", architecture: "x86_64", version: "3.10.0-862.6.3.el7.x86_64" }, platform: "CPython 3.6.7.final.0" }
2019-05-02T19:19:40.536-0700 I NETWORK  [conn9] end connection 127.0.0.1:35050 (3 connections now open)
2019-05-02T19:19:40.536-0700 I NETWORK  [conn8] end connection 127.0.0.1:35048 (2 connections now open)
2019-05-02T19:19:41.604-0700 I NETWORK  [conn10] received client metadata from 127.0.0.1:35056 conn10: { driver: { name: "PyMongo", version: "3.7.2" }, os: { type: "Linux", name: "Linux", architecture: "x86_64", version: "3.10.0-862.6.3.el7.x86_64" }, platform: "CPython 3.6.7.final.0" }
2019-05-02T19:19:41.606-0700 I NETWORK  [conn11] received client metadata from 127.0.0.1:35058 conn11: { driver: { name: "PyMongo", version: "3.7.2" }, os: { type: "Linux", name: "Linux", architecture: "x86_64", version: "3.10.0-862.6.3.el7.x86_64" }, platform: "CPython 3.6.7.final.0" }
2019-05-02T19:21:42.263-0700 I NETWORK  [conn11] end connection 127.0.0.1:35058 (3 connections now open)
2019-05-02T19:21:42.263-0700 I NETWORK  [conn10] end connection 127.0.0.1:35056 (2 connections now open)
2019-05-02T19:21:43.332-0700 I NETWORK  [conn12] received client metadata from 127.0.0.1:35064 conn12: { driver: { name: "PyMongo", version: "3.7.2" }, os: { type: "Linux", name: "Linux", architecture: "x86_64", version: "3.10.0-862.6.3.el7.x86_64" }, platform: "CPython 3.6.7.final.0" }
2019-05-02T19:21:43.333-0700 I NETWORK  [conn13] received client metadata from 127.0.0.1:35066 conn13: { driver: { name: "PyMongo", version: "3.7.2" }, os: { type: "Linux", name: "Linux", architecture: "x86_64", version: "3.10.0-862.6.3.el7.x86_64" }, platform: "CPython 3.6.7.final.0" }
2019-05-02T19:23:43.993-0700 I NETWORK  [conn13] end connection 127.0.0.1:35066 (3 connections now open)
2019-05-02T19:23:43.993-0700 I NETWORK  [conn12] end connection 127.0.0.1:35064 (2 connections now open)
2019-05-05T08:42:41.442-0700 I CONTROL  [signalProcessingThread] got signal 2 (Interrupt), will terminate after current cmd ends
2019-05-05T08:42:41.442-0700 I NETWORK  [signalProcessingThread] shutdown: going to close listening sockets...
